paddle/gserver/tests/test_LayerGrad.cpp

```diff
@@ -1802,6 +1802,25 @@ TEST(Layer, RowConvLayer) {
   }
 }
 
+TEST(Layer, PixelSoftmaxLayer) {
+  TestConfig config;
+  // config input_0
+  config.inputDefs.push_back({INPUT_DATA, "layer_0", 1024, 0});
+  LayerInputConfig* input = config.layerConfig.add_inputs();
+  ImageConfig* img = input->mutable_image_conf();
+  img->set_channels(4);
+  img->set_img_size(16);
+  img->set_img_size_y(16);
+
+  // config softmax layer
+  config.layerConfig.set_type("pixel_softmax");
+  config.layerConfig.set_name("pixelSofrmaxLayer");
+
+  for (auto useGpu : {false, true}) {
+    testLayerGrad(config, "pixel_softmax", 100, false, useGpu, true, 2);
```

Comment:
Do not use the 2 as epsilon.

```
testLayerGrad(config, "pixel_softmax", 100, false, useGpu, false);
```

If the `test_LayerGrad` is failed for `PixelSoftmaxLayer`, the reason is that the `+=` should be used when computing the gradient in the backward.